.github/workflows/CI-orchestrator.yml

```diff
@@ -0,0 +1,170 @@
+name: Orchestrator
+
+on:
+  pull_request:
```

Comment:
Probably we want to keep previous trigger:
```
on:
  pull_request:
    types: [opened, synchronize]
```
in order to avoid for example the orchestrator to run on `pull_request.updated` event (description changed, title changed or other non-code updates).

Reply:
Thanks for this, good spot!